### PR TITLE
OSD Ground Course and Cross Track Error fields

### PR DESCRIPTION
**Pull request by breadoven**
Support for https://github.com/iNavFlight/inav/pull/8604.

### CODE REVIEW

**Comment by Jetrell** [comment]
@breadoven I'm not sure if you're aware. A symbol for this function is already in there? `252`

**Comment by Jetrell** [comment]
I'm not sure if you're aware. A symbol for this function is already in there? `252`
@breadoven The reason its there. Is because it was added to the HD font. That was given to Caddx for the Avatar goggle. In anticipation of this feature being added one day. This goes for the 4 symbols in that location.

File: tabs/osd.js [hunk 1]
```diff
@@ -111,6 +111,8 @@ SYM.GLIDE_MINS = 0xD5;
 SYM.GLIDE_RANGE = 0xD4;
 SYM.FLIGHT_MINS_REMAINING = 0xDA;
 SYM.FLIGHT_DIST_REMAINING = 0x167;
+SYM.GROUND_COURSE = 0xDC;
+SYM.TRACK_ERROR = 0xDD;
```
Comment (MrD-RC) [inline tabs/osd.js:115]: Please use 0xFC for course track error symbol.

Comment (breadoven) [reply tabs/osd.js:115]: OK, missed those new icons. Added a few days ago by the looks of it ? Is there any reason they aren't included in `iNav character map.md` ? I take it 251, the mountain with an arrow sticking out of it, is terrain following ?

Comment (MrD-RC) [reply tabs/osd.js:115]: Not being in the .md file is my mistake. Yes, the mountain with and arrow is terrain following.

Comment (MrD-RC) [reply tabs/osd.js]: Yes, thank you 👍🏻

**Review by MrD-RC — APPROVED** [review]

**Pull request merged**